docs/source/tutorial/introduction.md

```diff
@@ -1,5 +1,6 @@
 ---
-title: 0. Introduction
+title: Introduction
```

Comment:
Does this line not need quotes because there aren't any spaces? Would quotes hurt?

Reply:
I honestly don't remember off hand which characters necessitate quotes. Maybe just _other_ quotes? In any case, quotes wouldn't hurt and I'll at least standardize the format across all these chapters.